Dockerfile

```diff
@@ -30,6 +30,8 @@ COPY --from=semgrep-cli /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 COPY --from=builder /src/bin /dist/bin
 
 COPY docs/ /docs/
+COPY auto.yaml /
```

Comment:
let's avoid having the file committed in the repo because it might break the license we are using